src/templates/stickyboard-simple/src/components/base/PageBase.js

```diff
@@ -68,6 +69,8 @@ class PageBase extends React.Component {
         this.board = React.createRef();
 
         this.state = {
+            //button Toggle
+            deleteButtonToggle: false, 
```

Comment:
I think you can use **isEditingMode** variable to show or hide delete button.
So, the additional variable like **deleteButtonToggle** is unnecessary.